### DISCUSSION TITLE
Monitoramento de links de revenda de e-commerce

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#374] [Discussão]
Opa pessoal beleza? Pintou um freela pra mim fazer aqui, mas não tenho muita noção de como, a cliente quer usar um link que a loja pra qual ela revende via facebook,redes sociais e contatos consiga computar as vendas que foram indicadas por ela.
Me falaram que com Zanox consigo isso. Mas nunca usei essa plataforma.
Com Google analytics e um link com utm_source também.
Só que não é confiavel porque caso exista um dropout e o cliente volte ao site por outra url, ou pelo google mesmo é uma venda que ela perdeu.
Alguem sabe uma solução para isso?
Obrigado.

**Comment by willycamargo** [comment]
Estou passando bem rápido por aqui, mas respondendo com a intenção de acompanhar e tentar ajudar:

Ao meu ver parece automação de marketing 🤔 
Ela usa alguma plataforma de automação e métricas?
O que conheço é:

RD Station (rdstation.com.br)
Hubspot (hubspot.com)
Kissmetrics (kissmetrics.com)
Pipz (pipz.com)

Mas dai é claro que algumas delas envolvem muito mais coisas do que apenas monitoramento de links.

**Comment by rubenmarcus** [comment]
@willycamargo  Então nao é bem minha especialidade também.

Estou quase indicando algum outro profissional.

Ela disse que tem orçamento a disposição para isso. Ela é uma revendedora de uma marca do Rio, que está embarcando em SP e quer ter relatório de quantas vendas no site foram feitas pelo link indicado por ela
Ela usa Iluria.

Pelo que eu vejo essas ferramentas envolvem muito mais que isso.
